src/Umbraco.Web.UI.Client/src/views/partialviewmacros/create.controller.js

```diff
@@ -52,23 +52,23 @@
                 }, function (err) {
 
                     vm.createFolderError = err;
-                    
+
                 });
             }
         }
 
         function createFile() {
-            $location.path("/developer/partialviewmacros/edit/" + node.id).search("create", "true");
+            $location.path("/settings/partialviewmacros/edit/" + node.id).search("create", "true");
```

Comment:
This will work, i think in the future we could stop hard coding the section and use `"/" + $routeParams.section + "/"` instead. All good for now, don't think we're going to allow moving this editor anytime soon.